examples/inference_server/server.py

```diff
@@ -0,0 +1,99 @@
+"""Flask inference server.
+
+Implements a Flask server that handles inference requests for some input via a HTTP handler.
+To run the server, run the following command from the root directory:
+`FLASK_APP=examples/inference_server/server.py flask run`
+"""
+
+import os
+import random
+import re
+import string
+import subprocess
+
+from flask import Flask, request, abort
+from werkzeug.utils import secure_filename
+import sky
```

Comment:
nit: please reorder the packages in alphabetic order.

Reply:
done! perhaps we could consider adding isort to `format.sh` to enforce this